x-pack/plugins/cases/common/api/connectors/mappings.ts

```diff
@@ -47,9 +47,6 @@ const ConnectorFieldRt = rt.type({
 
 export type ConnectorField = rt.TypeOf<typeof ConnectorFieldRt>;
 
-const GetFieldsResponseRt = rt.type({
```

Comment:
Did the response changed?

Reply:
we're not doing the dynamic fields anymore [since this PR](https://github.com/elastic/kibana/pull/97754), removed some artifact code instead of creating pointless code for Swimlane